config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -158,3 +206,107 @@ periodics:
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-azure
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190315-49d62eb51-master
+      args:
+      - --job=$(JOB_NAME)
+      - --repo=k8s.io/kubernetes=v1.14.0-alpha.1
+      - --repo=k8s.io/$(REPO_NAME)=master
+      - --root=/go/src
+      - --service-account=/etc/service-account/service-account.json
+      - --timeout=450
+      - --upload=gs://kubernetes-jenkins/logs/
+      - --scenario=kubernetes_e2e
+      - --
+      - --gce-ssh=
+      - --test=true
+      - --up=true
+      - --down=true
+      - --deployment=acsengine
+      - --build=bazel
+      - --provider=skeleton
+      - --ginkgo-parallel=30
+      - --test-ccm=True
+      - --acsengine-agentpoolcount=3
+      - --acsengine-admin-username=azureuser
+      - --acsengine-creds=$AZURE_CREDENTIALS
+      - --acsengine-orchestratorRelease=1.14
+      - --acsengine-mastervmsize=Standard_DS2_v2
+      - --acsengine-agentvmsize=Standard_D4s_v3
+      - --acsengine-ccm=True
+      - --acsengine-location=eastus2
+      - --acsengine-public-key=$AZURE_SSH_PUBLIC_KEY_FILE
+      - --acsengine-template-url=https://raw.githubusercontent.com/kubernetes/cloud-provider-azure/master/tests/k8s-azure/manifest/linux.json"
+      - --acsengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.31.2/aks-engine-v0.31.2-linux-amd64.tar.gz
+      - --timeout=420m
+      securityContext:
+        privileged: true
+      env:
+      - name: CCM_E2E_ARGS
+        value: "-ginkgo.focus=autoscaler"
+
+- interval: 8h
+  # ci-cloud-provider-azure-conformance runs Kubernetes conformance tests periodically.
+  name: ci-cloud-provider-azure-conformance
+  labels:
+    preset-service-account: "true"
+    preset-cloudprovider-azure-cred: "true"
+    preset-dind-enabled: "true"
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-azure
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-azure
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190315-49d62eb51-master
+      args:
+      - --job=$(JOB_NAME)
+      - --repo=k8s.io/kubernetes=v1.14.0-alpha.1
+      - --repo=k8s.io/$(REPO_NAME)=master
+      - --root=/go/src
+      - --service-account=/etc/service-account/service-account.json
+      - --timeout=450
+      - --upload=gs://kubernetes-jenkins/logs/
+      - --scenario=kubernetes_e2e
+      - --
+      - --gce-ssh=
+      - --test=true
+      - --up=true
+      - --down=true
+      - --deployment=acsengine
+      - --build=bazel
+      - --provider=skeleton
+      - --ginkgo-parallel=30
+      - --acsengine-agentpoolcount=3
+      - --acsengine-admin-username=azureuser
+      - --acsengine-creds=$AZURE_CREDENTIALS
+      - --acsengine-orchestratorRelease=1.14
+      - --acsengine-mastervmsize=Standard_DS2_v2
+      - --acsengine-agentvmsize=Standard_D4s_v3
+      - --acsengine-ccm=True
```

Comment:
Does this job require us to build a hyperkube image from `--repo=k8s.io/kubernetes=v1.14.0-alpha.1` or is it suffice to use whatever hyperkube image we provide in `--acsengine-template-url=https://raw.githubusercontent.com/kubernetes/cloud-provider-azure/master/tests/k8s-azure/manifest/linux.json` ? In theory the source of e2e tests, version of ccm and version of hyperkube should all match.

Reply:
I think building a new one is better, since the versions are updating frequently

Reply:
If this job needs to build a hyperkube image from whatever we set here --repo=k8s.io/kubernetes=v1.14.0-alpha.1` then you need to add `- "--acsengine-hyperkube=True"`. You might also need to add this flag in `pull-cloud-provider-azure-e2e` job since they are supposed to do the same. This also allows us to pass in `--repo=k8s.io/kubernetes=master` to build hyperkube from the latest commit.

Reply:
Good catch, fixed now in the new commit